.github/ci/common.sh

```diff
@@ -86,13 +86,19 @@ function release_binary {
       //src/app_charts:push \
       //src/go/cmd/setup-robot:setup-robot.push
 
+  # The push scripts depends on binaries in the runfiles.
+  local oldPwd=$(pwd)
```

Comment:
there was a reason to write this in two lines. I think something related to how errors are handled in bash. (Is it possible to get the review comments of my cl? I think there was a reference to the reason)

Reply:
I think the conversation history should still be there, but I cannot find any such thread on your PR. But in any case, I think you are right (there is a g3 linter often complaining about this), so I updated this.